### DISCUSSION TITLE
Trying to copy events into Google account's main calendar

### DISCUSSION THREAD

**Discussion by owljunior** [derekantrican/GAS-ICS-Sync#374] [Q&A] [answered]
Hi all. Love the script, super helpful!

I am trying to copy events from an ical into a Google account's main calendar. However, when I enter the title of the calendar into the script, instead of just copying events into the existing one, GAS-ICS-Sync creates a new calendar with the same name. That calendar ends up having the @group.calendar.google.com structure instead of the @google.com structure.

I tried forcing it to use Google account's main calendar ID by hard-coding it into the setupTargetCalendar function, but that did not work...

**Comment by derekantrican** [comment, marked as answer]
Use your gmail address instead of the calendar name. This is how Google identifies your primary calendar

**Reply by owljunior** [reply]
Aha, fantastic. Thank you!
